starport/pkg/cosmosgen/generate_typescript.go

```diff
@@ -206,20 +232,21 @@ func (g *jsGenerator) generateVuexModuleLoader() error {
 		})
 	}
 
-	loaderPath := filepath.Join(g.g.o.vuexStoreRootPath, "index.ts")
-
-	if err := templateVuexRoot.Write(g.g.o.vuexStoreRootPath, "", data); err != nil {
+	tsClientOut := filepath.Join(g.g.o.tsClientRootPath, "client")
```

Comment:
```suggestion
	tsClientOut := filepath.Join(g.g.o.tsClientRootPath, "ts-client")
```

Reply:
This _out_ will be dynamic so to have identical folder and npm-package names.